Sources/muterCore/TestReporting/HTML/HTMLReporter.swift

```diff
@@ -248,6 +248,7 @@ private extension MutationOperator.Id {
         case .ror: return "Relational Operator Replacement"
         case .removeSideEffects: return "Remove Side Effects"
         case .logicalOperator: return "Change Logical Connector"
+        case .ternaryOperator: return "Swap ternary operator expressions"
```

Comment:
I think a more appropriate name would be `Change Conditional Operator`.
At least it's what other mutation libs uses.